src/api/lib/db/client.js

```diff
@@ -2,24 +2,25 @@ import pg from 'pg';
 import coPg from 'co-pg';
 import named from 'node-postgres-named';
 
-export default function* dbClient({ host, port, user, password, database }) {
+export default async ({ host, port, user, password, database }) => {
     const postgresql = coPg(pg);
     const connection = `postgres://${user}:${password}@${host}:${port}/${database}`;
-    const connectionResponse = yield postgresql.connectPromise(connection);
-    const client = connectionResponse[0];
-    const done = connectionResponse[1];
+    const connectionResponse = await postgresql.connectPromise(connection);
+    const [client, done] = connectionResponse;
 
     named.patch(client);
-
     const query = client.query;
-
-    client.query_ = (queryString, values) =>
-        callback => query(queryString, values, callback);
-
-    client.id = (yield client.query_('SELECT pg_backend_pid()')).rows[0].pg_backend_pid;
+    client.queryPromise = (queryString, values) => new Promise((resolve, reject) => {
+        query(queryString, values, (err, result) => {
+            if (err) return reject(err);
+            return resolve(result);
+        });
+    });
+    const result = await client.queryPromise('SELECT pg_backend_pid()');
```

Comment:
I still don't understand this part... and IMO pools should be used instead, but this is not the aim of your PR :)